core/modules-manager.php

```diff
@@ -116,4 +131,8 @@ public function get_modules( $module_name ) {
 	protected function get_modules_namespace_prefix() {
 		return 'Elementor';
 	}
+
+	private function add_experimental_module( array $experimental_data ) {
```

Comment:
It's needed?

Reply:
It allows any module to set itself as an experiment with only calling the `get_experimental_data` method

Reply:
Done